cockroachdb/docs/backup.md

```diff
@@ -4,7 +4,7 @@ This runbook describes the backup procedure and how to restore from a backup.
 
 ## How to backup
 
-Using the manifests in this repo you have the option of setting a backup URL using the [configuration files](../examples/config/backup-config) and the [backup script](../base/scripts.yaml). This will automatically setup the backup schedule for you and if you have setup your s3 bucket and [vault injector](https://github.com/utilitywarehouse/documentation/blob/master/infra/vault/vault-aws.md) already the backups will all happen automatically on your schedule.
+Using the manifests in this repo you have the option of setting a backup URL using the [configuration files](../example/config/backup-config) and the [backup script](../manifests/scripts.yaml). This will automatically setup the backup schedule for you and if you have setup your s3 bucket and [vault injector](https://github.com/utilitywarehouse/documentation/blob/master/infra/vault/vault-aws.md) already the backups will all happen automatically on your schedule.
```

Comment:
There's a 404 link here (I think it was also 404-ing in the old repo). I've just guessed at where it's meant to be pointing at

```suggestion
Using the manifests in this repo you have the option of setting a backup URL using the [configuration files](../example/cfssl/config/backup-config) and the [backup script](../manifests/scripts.yaml). This will automatically setup the backup schedule for you and if you have setup your s3 bucket and [vault injector](https://github.com/utilitywarehouse/documentation/blob/master/infra/vault/vault-aws.md) already the backups will all happen automatically on your schedule.
```

Reply:
I will refactor the docs in separate PR